MVVM Module/Tests/Feed UI/Helpers/FeedViewController+TestHelpers.swift

```diff
@@ -3,7 +3,7 @@
 //
 
 import UIKit
-import MVVM
+@ testable import MVVM
```

Comment:
There's an extra space between @ and testable.

```suggestion
@testable import MVVM
```